docs/src/appendix/snforge-library/declare.md

```diff
@@ -2,6 +2,6 @@
 
 > `fn declare(contract: ByteArray) -> Result<ContractClass, Array<felt252>>`
 
-Declares a contract for later deployment.
+Declares a contract for later deployment. The `contract` is the name of the contract (part after `mod` keyword) e.g. `"HelloStarknet"`.
 
-See [docs of `ContractClass`](./contract_class.md) for more info about the resulting struct.
+Check docs of [`ContractClass`](./contract_class.md) for more info about the resulting struct.
```

Comment:
it's more visible with the fragment hyperlinked

Reply:
Imo it's better and more consistent this way

Reply:
Not a huge deal for me but please abstain from further correcting this, it's not worth the attention